example/package.json

```diff
@@ -10,7 +10,7 @@
     "react-dom": "^16.14.0",
     "react-live": "^2.2.3",
     "react-scripts": "^4.0.0",
-    "react-showtime": "https://github.com/azavea/react-showtime#develop",
+    "react-showtime": "link:..",
```

Comment:
It took a while to arrive back where we started here, but I think linking to the local version of `react-showtime` is the best overall approach. Since we've decided to use Git Flow, all merges to `main` will happen as part of a release process, and as such all commits to `main` will be tagged. The production example site will effectively always use the most recent stable release of `react-showtime`.

Going with this approach means we don't have to worry about starting a hotfix branch (or another release) to apply updates to the example site post-release and we don't need to worry about continually bumping `example/yarn.lock`. It also cuts the size of the release checklist in half and takes the manual `link` steps out of local development. This satisfies all of our requirements given our workflow.